app/views/admin/blog_posts/index.html.erb

```diff
@@ -45,6 +45,8 @@
             </tr>
           <% end %>
         </table>
+        <% scope = BlogPost.all.page(params[:page]) %>
+        <%= paginate(scope) %>
```

Comment:
You should use the instance variable here (`@blog_posts`) since that's what you want to paginate.